architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -131,21 +135,27 @@ Hub] as an example implementation of a Custom builder image.
 [[pipeline-build]]
 === Pipeline Build
 
-The Pipeline build strategy allows developers to define a Jenkins pipeline 
-which will be executed by the Jenkins pipeline plugin.  The build can still be
-started, monitored, and managed from within {product-title} in the same way as any other build
-type.
+The Pipeline build strategy allows developers to define a _Jenkins pipeline_
+for execution by the Jenkins pipeline plugin.
+The build can be started, monitored, and managed by
+{product-title} in the same way as any other build type.
+
+Pipeline workflows are defined in a Jenkinsfile,
+either embedded directly in the build configuration,
+or supplied in a Git repository and referenced by the build configuration.
 
-Pipeline workflows are defined in a Jenkinsfile which can either be embedded directly
-in the build configuration, or supplied in a git repository referenced by the build
-configuration.
+The first time a project defines a build configuration using a Pipeline strategy,
+{product-title} instantiates a Jenkins server to execute the pipeline.
+Subsequent Pipeline build configurations
+in the project share this Jenkins server.
 
-When a build configuration that uses a Pipeline strategy is first defined within a project,
-a Jenkins server will be instantiated to execute the pipeline.  All pipeline build configurations
-within a single project will share the Jenkins server.  The server is not automatically removed even 
-if all pipeline build configurations are deleted, it must be deleted by the user.
+[NOTE]
+The Jenkins server is not automatically removed
+even if all Pipeline build configurations are deleted.
+It must be deleted by the user.
```

Comment:
The `====` are optional in the case of a single paragraph admonition.

+1 to the "it must" rewording (see https://github.com/openshift/openshift-docs/pull/2549/commits/8ef7df0f162c57a5691095df1042bf585bdac932).